tractor/_actor.py

```diff
@@ -162,6 +170,14 @@ class Actor:
     _root_nursery: trio.Nursery
     _server_nursery: trio.Nursery
 
+    # marked by the process spawning backend at startup
+    # will be None for the parent most process started manually
+    # by the user (currently called the "arbiter")
+    _spawn_method: Optional[str] = None
```

Comment:
ditto. Unless this has different semantics which is to just choose whichever was chosen in `run`, in which case the Optional would be correct.

Reply:
@salotz for the parent-most (called arbiter) right now this will actually stay as `None` since there was no *spawn method* that created the top most process.